src/dsymbol/symbol.d

```diff
@@ -424,6 +424,30 @@ public:
 
 }
 
+/**
+ * istring with actual content and information if it was ditto
+ */
+struct DocString
+{
+	/// creates a non-ditto comment
+	this(istring content)
+	{
+		this.content = content;
+	}
+
+	/// creates a comment which may have been ditto
+	this(istring content, bool ditto)
+	{
+		this.content = content;
+		this.ditto = ditto;
+	}
+
+	alias content this;
+
+	istring content;
```

Comment:
can the two fields be `immutable` or `const` ? You only modify them in a ctor.

Reply:
Adding immutable or const broke a lot of code because reassigning a DocString doesn't work then

```
dsymbol/src/dsymbol/conversion/first.d(356): Error: generated function `dsymbol.symbol.DocString.opAssign` cannot be used because it is annotated with @disable
dsymbol/src/dsymbol/conversion/first.d(787): Error: generated function `dsymbol.symbol.DocString.opAssign` cannot be used because it is annotated with @disable
dsymbol/src/dsymbol/conversion/first.d(787): Error: generated function `dsymbol.symbol.DocString.opAssign` cannot be used because it is annotated with @disable
dsymbol/src/dsymbol/conversion/first.d(835): Error: generated function `dsymbol.symbol.DocString.opAssign` cannot be used because it is annotated with @disable
dsymbol/src/dsymbol/conversion/first.d(860): Error: generated function `dsymbol.symbol.DocString.opAssign` cannot be used because it is annotated with @disable
```

Reply:
okay then